cmd/admission-server/main.go

```diff
@@ -80,7 +80,9 @@ func main() {
 	<-signalChan
 
 	glog.Infof("Got OS shutdown signal, shutting down webhook server gracefully...")
-	wh.Server.Shutdown(context.Background())
+	if err := wh.Server.Shutdown(context.Background()); err != nil {
```

Comment:
This way of error handling is not idiomatic.
```
err:=someFunc()
if err!=nil{
...
}
```

Reply:
changes done sir.